scripts/lib/contentful/update-template.ts

```diff
@@ -16,7 +16,7 @@ export default async function updateTemplate({
   const readme = await getReadme(examplePath)
```

Comment:
Here's the top level I was referring to in my previous comment

Reply:
I can add that, however, we weren't catching it before this PR either.

Reply:
Actually, do we want to catch it? We want the script to fail, so I don't think we do.

Reply:
ahh okay fair enough! In that case let's merge it! :)